app/src/test/java/com/example/moviesapp/viewmodel/MainActivityViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.moviesapp.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.example.moviesapp.testObserver
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class MainActivityViewModelTest {
+
+    private lateinit var mainActivityViewModel: MainActivityViewModel
+
+    @get:Rule
+    var rule: TestRule = InstantTaskExecutorRule()
```

Comment:
do you need this? do you have any coroutine?

Reply:
No, fixed

Reply:
this still here